sql/src/SqlEval.sk

```diff
@@ -149,75 +149,84 @@ fun computeInsert(
         val,
       )
     });
-    schema = table.schema;
-    schemaSize = schema.size();
-    valueSize = values.size();
-    cvalues = columnParamsOpt match {
-    | None() ->
-      if (valueSize != schemaSize) {
-        error(pos, "Value size mismatch with table schema");
-      };
-      values
-    | Some(columnParams) ->
-      missingCols = mutable Map[];
-      for (col in schema) {
-        missingCols![col.name] = col;
-      };
-      for (userDefinedCol in columnParams) {
-        if (!missingCols.containsKey(userDefinedCol)) {
-          error(pos, "Column not found: " + userDefinedCol);
-        };
-        missingCols.remove(userDefinedCol);
+    computeInsertRow(context, pos, columnParamsOpt, values, table, repeat)
+  })
+}
+
+fun computeInsertRow(
```

Comment:
Github does not show the diff well, but note that the body of this function is simply moved unchanged. No need to carefully compare.

Reply:
Thanks. Very helpful. :)